webclient/app/src/app/config/DataTypes.js

```diff
@@ -0,0 +1,42 @@
+const DataTypes = [
```

Comment:
Die Datentypen stimmen nicht mit dem Backend überein.

Reply:
Ich nehme an, dass die in https://dev.azure.com/starwit/lj-projectbuilder/_boards/board/t/lj-projectbuilder%20Team/Backlog%20items/?workitem=105 stimmen?